apps/backoffice-v2/src/domains/workflows/utils/get-origin-url.ts

```diff
@@ -0,0 +1,5 @@
+export const getOriginUrl = (link: string) => {
```

Comment:
Why is this needed?

Reply:
This helper needed to extract host name of api url so i can build link to any other endpoint excpect `api/v1/internal` as it is now. Currently this is somewhat issue, all endpoinds are binded to /internal endpoints. We should update all fetchers accordingly to endpoint destination.

Reply:
Isn't this solved by the case management backend module?

Reply:
Nope, you cant reach it by using `internal` path piece.